docs/integration_resources/resources/usd_tutorials/usd_tutorial_02.md

```diff
@@ -0,0 +1,205 @@
+
+---
+
+<p style="text-align:left;">
+
+[< Creating a Simple USD Stage](usd_tutorial_01.md)
+<span style="float:right;">
+[Referencing Layers >](usd_tutorial_03.md)
+</span>
+
+</p>
+
+---
+
+## Inspecting and Authoring Properties
+
+This tutorial focuses on inspecting the properties containing geometric data of the prims created in the tutorial [Creating a Simple USD Stage](usd_tutorial_01.md), using the operators from the Shift_USD catalog.
+
+The base file for the example is the resulting USD from the previous tutorial named `HelloWorld.usda`. Also, it can be downloaded from the [OpenUSD](https://github.com/PixarAnimationStudios/OpenUSD) Github project repository, from `extras/usd/tutorials/authoringProperties/` folder.
```

Comment:
I was thinking about this too. Because if we provide the solution to the tutorials, maybe the users will try to directly use them as input data for the workflow, and that won't work as expected because those files will be the solution already.
I added the starting USD files from the tutorials next to the provided workflows for implementing each one of the tutorials, just in case, they will have the input data next to the workflow of the tutorial.
What maybe can be useful is to add a new folder named `result`, and inside we can place the USD files, to separate between input data for the workflows and output data. We can discuss this during the SCRUM if you prefer. And after confirming, we can address this one. Thanks for pointing this!

Reply:
Yes, let's mention it in the scrum